TheMovieDb/Scenes/Review/AuthorView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AuthorView.swift
+//  TheMovieDb
+//
+//  Created by Juan Alfredo García González on 30/11/21.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct AuthorView: View {
+    
+    private let baseUrl = "https://image.tmdb.org/t/p/w185"
+    var author: AuthorDetail?
+    
+    var body: some View {
+        HStack() {
+            let urlString = "\(baseUrl)\(author?.avatarPath ?? "")"
```

Comment:
Although is encouraged to have logic-less views, you can use computed values like:
`private var urlString: String { "\(baseUrl)\(author?.avatarPath ?? "")" }`

that way your declarative body view is clearer to read